packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5453,6 +5453,34 @@ class Flow extends MultiChildRenderObjectWidget {
 /// ```
 /// {@end-tool}
 ///
+/// ## Selections
+///
+/// To make this [RichText] Selectable, the [RichText] needs to be in the
+/// subtree of a [SelectionArea] or [SelectableRegion] and assigned a
+/// [SelectionRegistrar] to the [RichText.selectionRegistrar]. One can use
```

Comment:
"and assigned a [SelectionRegistrar] to the" => "and a [SelectionRegsitrar] needs to be assigned to the"